.github/workflows/check-links.yml

```diff
@@ -7,15 +8,22 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
Allows user to trigger CI from github UI for  a given branch.

![image](https://github.com/atorus-research/xportr/assets/211358/9dce6b06-4777-47ff-83ce-31e31a16b2be)